Apps/Playground/Scripts/config.json

```diff
@@ -206,6 +206,17 @@
             "playgroundId": "#BCYE7J#31",
             "referenceImage": "multiCamerasOutputRenderTarget.png"
         },
+        {
+            "title": "Multi camera rendering with scissors",
+            "playgroundId": "#1LK70I#35",
+            "referenceImage": "multiCameraRenderingWithScissors.png"
+        },
+        {
+            "title": "Multi cameras and output render target with scissors",
+            "renderCount": 2,
+            "playgroundId": "#BCYE7J#45",
+            "referenceImage": "multiCamerasOutputRenderTargetWithScissors.png"
+        },
```

Comment:
Are these native specific? I don't think we have any visual tests that are native specific. Ideally, there shouldn't be any.

Reply:
Ok. Babylon.js has one vis test for scissor we can bring over. Should I remove the two I added here or put them in the Babylon.js vis test list?

Reply:
It depends if the coverage is good or not. Is the Babylon.js one covering the same things?

Reply:
The Babylon.js vis test covers the same things as the 2nd test I added here named "Multi cameras and output render target with scissors". I'll replace it with the JS test.

I think we should keep the 1st test I added here, though, since it has viewports and multiple cameras, too, which can create some unexpected interactions if we're not careful. Should I add this 1st test back to the Babylon.js tests, too? ...or do we call this a native-specific test and leave it out of the JS tests?

Reply:
> Should I add this 1st test back to the Babylon.js tests, too?

I think so. I don't see a reason why there should be a difference in testing. If it makes sense to test in native, it should also make sense to test in JS.

Reply:
I think some of the tests I added here will only test browser implementations in JS, not Babylon.js itself. I don't think it makes sense to add all of them to the JS tests.

For example, the out of bounds tests I added today don't make much sense to add to the JS tests since Babylon.js just passes the scissor parameters directly to the WebGL implementation.

Reply:
If that's the case, we need to come up with a mechanism for Babylon Native specific tests. Otherwise, it will get more difficult to maintain what tests go where.

Reply:
So the current config.json would be just for tests that apply to both native and JS, and then add another config file for native-only tests, named something like config-native.json?

Reply:
I don't know. Let's discuss offline.

Reply:
I asked Seb if he wants the new tests in Babylon.js and he's cool with it, so there's no need to have separate that are specific to Babylon Native, yet.